eclass/check-reqs.eclass

```diff
@@ -43,16 +43,25 @@ case ${EAPI} in
 	*) die "${ECLASS}: EAPI=${EAPI:-0} is not supported" ;;
 esac
 
-EXPORT_FUNCTIONS pkg_pretend pkg_setup
-
 if [[ ! ${_CHECK_REQS_ECLASS} ]]; then
 _CHECK_REQS_ECLASS=1
 
+inherit multiprocessing
+
+EXPORT_FUNCTIONS pkg_pretend pkg_setup
```

Comment:
`EXPORT_FUNCTIONS` should be probably outside inclusion guard (`if [[ ! ${_CHECK_REQS_ECLASS} ]]; then`) (and still after any calls to `inherit`).

https://gitweb.gentoo.org/repo/gentoo.git/log/eclass?qt=grep&q=EXPORT_FUNCTIONS
https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=f24a4018a32e3cc7bf6017ccb944574acb7f4727